test/inductor/test_perf.py

```diff
@@ -343,6 +343,20 @@ def f(a, b, c):
         inp = (T(10, 10), T(10, 10), T(10, 10))
         self.assertExpectedInline(count_numel(f, *inp), """500""")
 
+    def test_cat_pointwise_inps(self):
```

Comment:
If these are the only tests I want to see some more :P Cover the different cases.

Reply:
There are many existing cat tests in this file. But I can add more.